platform/android/src/geojson/polygon.cpp

```diff
@@ -29,6 +29,7 @@ mapbox::geojson::polygon Polygon::convert(jni::JNIEnv &env, jni::Object<java::ut
         for (auto&& line : multiLine) {
             polygon.emplace_back(convertExplicit<mapbox::geojson::linear_ring>(std::move(line)));
         }
+        jni::DeleteLocalRef(env, multiLine);
```

Comment:
I think `multiLine` should be `jPointListsList`

Reply:
@tobrun `jPointListsList` is a parameter in that function so caller will DeleteLocalRef to it

Reply:
yes 🤦‍♂️